docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3064/3064_2.md

```diff
@@ -0,0 +1,38 @@
+# Optionally hide maintenance configuration for pipeline jobs
+
+Test verifies that maintenance configuration for pipeline can be optionally hidden.
+
+**Prerequisites**:
+- Admin user
+- Perform [3064_1](3064_1.md) case
+
+**Preparations**:
+1. Login as the admin user from the prerequisites
+2. Create pipeline
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Click the **PREFERENCES** tab | |
+| 4 | Set value <ul> `{` <ul> `"pause": false` </ul> `}` </ul> for **`ui.run.maintenance.pipeline.enabled`** preference  | |
+| 5 | Click **Save** button | |
+| 6 | Open the **Library** page | |
+| 7 | Open pipeline created at step 2 of Preparations | |
+| 8 | Click the **Run** button | |
+| 9 | Expand the "**Advanced**" section | |
+| 10 | Select **_On-demand_** value for **Price type** | **Maintenance** control appears under the **Price type** |
+| 11 | Click **_Configure_** button for **Maintenance** control | **Maintenance** pop up appears |
+| 12 | Click the **Add rule** button | <li> The first rule appears <li> Only `Resume` action is available |
+| 13 | Launch the run | |
+| 14 | At the **Runs** page, click the just-launched run | **Maintenance** control is shown on the run details page |
+| 15 | Repeat 11-12 steps | Only `Resume` action is available |
+| 16 | Repeat steps 2-3 | | 
+| 17 | Set value <ul> `{` <ul> `"pause": false,` <br> `"resume": false` </ul> `}` </ul> for **`ui.run.maintenance.pipeline.enabled`** preference  | |
+| 18 | Click **Save** button | |
+| 19 | Repeat steps 6-10 | **Maintenance** control doesn't appear under the **Price type** |
+| 20 | Repeat steps 13-14 | **Maintenance** control isn't shown on the run details page |
+
+**After:**
+- Stop runs launched at step 14 of [3064_1](3064_1.md) case and at step 13 of [3064_2](3064_2.md) case
```

Comment:
> ... at step 14 of [3064_1](3064_1.md) ...

Fix to "at steps 14, 21 of [3064_1](3064_1.md)"

> ...at step 13 of [3064_2](3064_2.md)...

Fix to "at steps 13, 20 of [3064_2](3064_2.md)"